internal/store/builder.go

```diff
@@ -210,115 +212,124 @@ func availableResources() []string {
 }
 
 func (b *Builder) buildConfigMapStore() cache.Store {
+	// no labels
```

Comment:
The idea was to make reviews easier and future changes to this function so it easily shows why this call is  different to other ones that do have label metrics. I was planning on removing it but maybe rephrasing it to something like:
```
// There are no labels metrics for this resource.
```
Would that be helpful, or should I simply remove the comment?

I would not make them to TODOs as if there is no user need to add those metrics. Just because we can, doesn't mean we should if no one is interested in them. With that said if someone adds them we will accept it. Hope that makes sense?

Reply:
I might have misunderstood, I thought you wanted to add all the remaining labels metrics for completeness purposes.

Considering your previous statement, I agree that it's not really meaningful to have them as TODOs and I am fine with removing the comments completely.

Reply:
Sorry was not clear, it makes sense to have them, but there is nothing broken if they are not there, and as soon as we get an actual use case for it we can add each one. For completeness sake I would not be adding it if folks won't make use of them. Some things like ConfigMaps make more sense than other resources.

Reply:
Removed, thanks :)